openedx/openedx-platform
# [ENT-494] Move to new consent API

This pull request updates edx-platform to use the new data sharing consent API.

**JIRA tickets**: [ENT-494](https://openedx.atlassian.net/browse/ENT-494)

**Dependencies**: edx/edx-enterprise#158

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: 14 August 2017

**Testing instructions**:

1. Enroll a user in a course via the Enterprise admin panel
2. Ensure your enterprise is set to require data sharing consent
3. Log in with the user in question and attempt to access the course
4. Verify that you're prompted for consent; accept.
5. Observe that you're brought to the course content you want.
6. Navigate away from the course and back to it.
7. Verify that you are not prompted for consent.
8. Enroll yourself in another course (non-Enterprise)
9. Observe that when you enter the course, you are not prompted for consent.
10. Go through the Enterprise landing page flow, noting that you are prompted for consent, and that after giving it, you are able to access the course.

**Author notes and concerns**:

This is very much in progress.

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
